core/cog-directory-files-handler.c

```diff
@@ -186,6 +188,27 @@ cog_directory_files_handler_run (CogRequestHandler      *request_handler,
     while (path[0] == '/')
         ++path;
 
+    /*
+     * Discard non-empty leading path components, ignoring empty path
```

Comment:
Very appreciate this comment. It makes much more simple the understanding of the code. Thanks.